README.md

```diff
@@ -38,6 +38,14 @@ make get-localic
 # Attempt to run a command
 spawn help
 
+# Potential workaround if spawn is immediately killed when attempting run "spawn help" on a m* macbook. 
+# This may happen because of an xcode16 update.
+# To work around this as a temporary measure please change this line in the Make file. 
+LD_FLAGS = -X main.SpawnVersion=$(VERSION)
+# to 
```

Comment:
Can you instead put this in the mainfile itself?

We can probably just use those flags for every install on any machine. A reason why we use them too would be nice as well